Domain.CardGamesGuruMiniApp/Mapping/EnumMapping.cs

```diff
@@ -16,12 +16,12 @@ public static GameType MapGameType(string gameType)
 
             switch (gameType)
```

Comment:
Don't use `switch`. You can use [`Enum.TryParse`](https://learn.microsoft.com/ru-ru/dotnet/api/system.enum.tryparse?view=net-7.0)